src/nvim/buffer_defs.h

```diff
@@ -936,8 +934,8 @@ struct matchitem {
  * All row numbers are relative to the start of the window, except w_winrow.
  */
 struct window_S {
-  uint64_t handle;
-  int w_id;                         ///< unique window ID
+  handle_T handle;                  ///< unique identifier for the window
+
```

Comment:
You mean the PR itself, not when basing extmark on it? Does `rm -rf build` help?